can/interfaces/slcan.py

```diff
@@ -287,7 +287,9 @@ def get_version(
 
         string = self._read(timeout)
 
-        if len(string) == 6 and string[0] == cmd:
+        if not string:
```

Comment:
Added back the `None` checking.
Due to the unit test using the `loop://` parameter. It can not catch the `None` error.